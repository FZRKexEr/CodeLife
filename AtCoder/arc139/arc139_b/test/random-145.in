100
84 5 14 7 9 20
76 7 1 19 16 1
80 11 16 19 13 19
73 4 5 20 12 8
34 8 14 20 6 18
46 15 9 5 2 7
64 18 17 1 9 8
83 12 20 3 2 12
100 15 4 14 5 5
93 2 15 3 14 9
93 15 14 2 15 2
54 18 12 5 18 13
34 14 19 16 12 19
83 11 10 14 10 10
85 18 9 13 2 17
93 13 17 15 15 3
64 11 14 16 3 20
13 19 14 18 1 7
11 17 1 6 19 7
58 11 20 18 18 11
54 9 12 15 2 16
32 11 13 19 16 14
14 18 5 1 11 10
37 10 5 4 13 3
13 13 19 5 2 17
62 3 14 14 16 7
19 13 5 15 13 14
11 14 15 2 13 19
53 19 6 15 4 20
72 3 2 13 15 3
69 14 6 20 10 5
7 3 8 19 9 15
78 20 11 7 19 11
88 5 17 9 17 4
72 2 17 1 14 2
49 2 5 9 13 20
72 19 7 7 2 7
13 17 6 9 16 1
77 5 16 4 4 12
83 16 10 13 10 8
23 1 9 5 12 8
79 10 20 6 15 17
13 15 5 19 8 4
46 10 15 3 10 11
70 17 3 9 12 11
3 13 2 14 16 15
66 12 17 14 17 5
62 12 16 19 11 16
65 2 11 12 8 18
83 1 17 2 16 17
25 4 16 6 4 19
13 18 13 7 8 8
81 13 19 3 3 4
48 13 18 10 11 15
35 3 12 17 7 6
98 18 7 15 7 6
88 5 14 1 11 6
61 17 13 6 5 7
29 8 10 12 1 9
90 3 2 2 15 17
94 7 13 8 1 20
12 8 3 18 12 8
80 19 17 2 9 1
40 19 15 7 9 3
57 11 14 11 9 1
43 15 9 12 4 4
7 8 5 1 2 14
29 4 16 11 8 1
32 1 13 7 10 20
27 6 7 8 4 2
15 3 17 6 5 14
77 7 11 4 16 17
81 16 14 1 5 18
44 17 15 13 10 12
40 10 5 8 20 4
12 11 11 4 10 15
2 15 1 18 7 16
58 7 4 5 14 6
10 15 18 3 8 6
9 9 20 7 10 6
2 9 3 20 7 1
56 16 2 13 4 2
46 5 9 9 5 5
92 13 8 1 1 15
47 3 13 7 16 15
69 1 11 3 10 6
56 15 18 4 13 18
43 7 15 14 4 17
3 7 20 5 6 1
100 3 11 3 12 7
50 18 18 17 5 20
92 7 3 3 19 12
55 13 14 1 14 12
55 19 4 5 16 20
59 4 18 19 14 13
77 17 12 19 7 5
38 18 18 8 5 14
23 6 11 3 12 5
58 8 14 17 11 3
94 11 10 12 15 19
