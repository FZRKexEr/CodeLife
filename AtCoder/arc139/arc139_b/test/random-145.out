120
76
95
128
64
11
37
31
68
63
30
54
47
80
77
99
66
234
66
144
12
142
14
29
2
60
42
22
93
108
70
37
84
47
12
206
27
11
60
73
41
120
65
36
66
29
74
93
130
98
30
8
27
77
67
56
40
30
31
180
37
32
28
32
15
28
7
36
82
10
25
119
26
150
32
14
32
94
30
10
40
20
30
8
91
45
62
33
15
66
248
250
49
117
72
32
26
13
31
155
